lib/wrapped_print.rb

```diff
@@ -14,6 +18,9 @@ module WrappedPrint
   mattr_accessor :log_to
   @@log_to = :console
 
+  mattr_accessor :custom_logger
```

Comment:
I think you can use log_to variable

I mean if you want to log to file

```
    WrappedPrint.setup do |config|
      config.log_to = ActiveSupport::Logger.new("some/file.log")
    end
```

no need to specify "custom_logger" and have a fallback to  ActiveSupport::Logger.new("log/wrapped_print.log")